sdk/include/psdk/shlobj.h

```diff
@@ -1649,30 +1651,43 @@ typedef enum RESTRICTIONS
 	REST_NOLOCALMACHINERUNONCE,
 	REST_NOCURRENTUSERRUNONCE,
 	REST_FORCEACTIVEDESKTOPON,
-	REST_NOCOMPUTERSNEARME,
-	REST_NOVIEWONDRIVE,
-	REST_NONETCRAWL,
-	REST_NOSHAREDDOCUMENTS,
-	REST_NOSMMYDOCS,
-	REST_NOSMMYPICS,		/* 0x40000050 */
+#if (NTDDI_VERSION < NTDDI_VISTA)
```

Comment:
Question to reviewers: Do you think Wine would be OK with all this extra versioning information?
(Btw. MS PSDK does have it. We and Wine didn't so far.)

Reply:
Currently, Wine doesn't do multi-target versioning AFAIK.
https://github.com/wine-mirror/wine/search?q=_WINVER&type=code
https://github.com/wine-mirror/wine/search?q=_WIN32_WINNT&type=code
https://github.com/wine-mirror/wine/search?q=NTDDI_VERSION&type=code

Reply:
OK; but maybe it could be of interest for us (regarding the compatibility layers...) 🤔

Reply:
I should also note that the extra versioning information has been added in the MS PSDK headers starting the Windows 7 PSDK. Beforehand, there were no versioning information for this `RESTRICTIONS` enumeration in the PSDK of Windows XP / 2003.